db/migrate/20220325135007_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
+
```

Comment:
- [ ] bio may store large text, let's give it `text` as data type. This is the more reason why on the ERD it has been given `text`.

Reply:
Done

Reply:
You can apply this to the `: posts` and `:comments`

The posts should have `t.text :text` not a string
The comments should have `t.text :text` not a string

Reply:
Thanks for the review. I have made the required changes. Thanks